openedx/features/calendar/api.py

```diff
@@ -0,0 +1,19 @@
+"""
+API for calendar syncing Course dates with a User.
```

Comment:
Is it our custom to hide these functions inside an api.py? That's fine, but wouldn't we want to import the names into __init__ or something, so users don't have to import from an api file? Seems redundant from their POV.